docs/compatibility.md

```diff
@@ -80,3 +80,9 @@ model-level compatibility but slightly improves the performance. The major ones
 To convert the models trained by MMDetection V1.x to MMDetection V2.0, the users can use the script `tools/model_converters/upgrade_model_version.py` to convert
 their models. The converted models can be run in MMDetection V2.0 with slightly dropped performance (less than 1% AP absolute).
 Details can be found in `configs/legacy`.
+
+## pycocotools compatibility
+
+Before [PR 4939](https://github.com/open-mmlab/mmdetection/pull/4939), since `pycocotools` and `mmpycocotool` have the same package name, if users already installed `pyccocotools` (installed Detectron2 first under the same environment), then the setup of MMDetection will skip installing `mmpycocotool`. Thux MMDetection fails due to the missing `mmpycocotools`.
```

Comment:
Indicate that `mmpycocotools` works for both MMDet and Detectron2, so it will be fine if MMDetection is installed before Detectron2.